### PR TITLE
Release/v0.0.4

### PR DESCRIPTION
**Pull request by damienbfs**

### CODE REVIEW

**Comment by PaulineMauryL** [comment]
## Coverage report


 <details><summary>Click to see where and how coverage changed</summary><table><thead>
  <tr><th>File</th><th>Statements</th><th>Missing</th><th>Coverage</th><th>Coverage<br>(new stmts)</th><th>Lines missing</th></tr>
</thead>
<tbody><tr>
<td colspan="6">&nbsp;&nbsp;<b>server/lomas_server</b></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-4ef5cc30e18c28ff2745a2889f8b0ff899cca1f8bbf06fc33d0333dde1f48465">constants.py</a></td>

<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-4ef5cc30e18c28ff2745a2889f8b0ff899cca1f8bbf06fc33d0333dde1f48465"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-4ef5cc30e18c28ff2745a2889f8b0ff899cca1f8bbf06fc33d0333dde1f48465"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-4ef5cc30e18c28ff2745a2889f8b0ff899cca1f8bbf06fc33d0333dde1f48465"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-4ef5cc30e18c28ff2745a2889f8b0ff899cca1f8bbf06fc33d0333dde1f48465"></a></td><td></td><tr>
<td colspan="6">&nbsp;&nbsp;<b>server/lomas_server/dp_queries/dp_libraries</b></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-fa24a03009808c4b8ce9362d10c36fdcdaa1b1043361cfd8964094bee91c1ecb">opendp.py</a></td>

<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-fa24a03009808c4b8ce9362d10c36fdcdaa1b1043361cfd8964094bee91c1ecb"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-fa24a03009808c4b8ce9362d10c36fdcdaa1b1043361cfd8964094bee91c1ecb"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-fa24a03009808c4b8ce9362d10c36fdcdaa1b1043361cfd8964094bee91c1ecb"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-fa24a03009808c4b8ce9362d10c36fdcdaa1b1043361cfd8964094bee91c1ecb"></a></td><td><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-fa24a03009808c4b8ce9362d10c36fdcdaa1b1043361cfd8964094bee91c1ecbR146-R152">146-152</a>, <a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-fa24a03009808c4b8ce9362d10c36fdcdaa1b1043361cfd8964094bee91c1ecbR156-R161">156-161</a></td><tr>
<td colspan="6">&nbsp;&nbsp;<b>server/lomas_server/private_dataset</b></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-63f32a3e50743f125d1d2fffa0e91ccb35e89c98c53cdb69f5838fbb8398e8af">utils.py</a></td>

<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-63f32a3e50743f125d1d2fffa0e91ccb35e89c98c53cdb69f5838fbb8398e8af"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-63f32a3e50743f125d1d2fffa0e91ccb35e89c98c53cdb69f5838fbb8398e8af"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-63f32a3e50743f125d1d2fffa0e91ccb35e89c98c53cdb69f5838fbb8398e8af"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-63f32a3e50743f125d1d2fffa0e91ccb35e89c98c53cdb69f5838fbb8398e8af"></a></td><td></td><tr>
<td colspan="6">&nbsp;&nbsp;<b>server/lomas_server/tests</b></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-7f7d18385b324497d5ebcf51c3e0c0172158683cd7f916a75a261815c51f617c">test_api.py</a></td>

<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-7f7d18385b324497d5ebcf51c3e0c0172158683cd7f916a75a261815c51f617c"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-7f7d18385b324497d5ebcf51c3e0c0172158683cd7f916a75a261815c51f617c"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-7f7d18385b324497d5ebcf51c3e0c0172158683cd7f916a75a261815c51f617c"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-7f7d18385b324497d5ebcf51c3e0c0172158683cd7f916a75a261815c51f617c"></a></td><td></td><tr>
<td colspan="6">&nbsp;&nbsp;<b>server/lomas_server/utils</b></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-517d7a2783eedf30c47a796fd7e5f657d815318c06656e3b57bf6ad2a5dc39f7">utils.py</a></td>

<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-517d7a2783eedf30c47a796fd7e5f657d815318c06656e3b57bf6ad2a5dc39f7"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-517d7a2783eedf30c47a796fd7e5f657d815318c06656e3b57bf6ad2a5dc39f7"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-517d7a2783eedf30c47a796fd7e5f657d815318c06656e3b57bf6ad2a5dc39f7"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-517d7a2783eedf30c47a796fd7e5f657d815318c06656e3b57bf6ad2a5dc39f7"></a></td><td></td></tbody>
<tfoot>
<tr>
<td><b>Project Total</b></td>

<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-4b0bf2efa3367c0072ac2bf1e234e703dc46b47aaa4fe9d3b01737b1a15752b1"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-4b0bf2efa3367c0072ac2bf1e234e703dc46b47aaa4fe9d3b01737b1a15752b1"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-4b0bf2efa3367c0072ac2bf1e234e703dc46b47aaa4fe9d3b01737b1a15752b1"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/105/files#diff-4b0bf2efa3367c0072ac2bf1e234e703dc46b47aaa4fe9d3b01737b1a15752b1"></a></td><td>&nbsp;</td>
</tr>
</tfoot>
</table>

<sub>

This report was generated by [python-coverage-comment-action](https://github.com/py-cov-action/python-coverage-comment-action)

</sub>
</details>

File: docs/source/github_workflow.rst [hunk 1]
```diff
@@ -1,37 +1,91 @@
...
-* master
-* release
+* **master**: This is the stable branch. Release tags are always on this branch and the latest release is always the head of this branch.
+* **develop**: This is the main development branch and can be ahead of the master branch.
+  One should never directly merge and push to develop but perform a pull request on GitHub.
+  The PR can only be merged if approved by another developer and all automatic tests pass.
+* **wip_xx**: Feature branches for feature number xx start with wip_xx (one can add a short name to the branch name).
+  They always branch off develop, and as explained above, are merged to develop via GitHub pull requests.
+* **release/vx.y.z**: These are release branches (for version vx.y.z). They always branch off from develop.
+  Once the release process is complete (see below), the release branch is merged to both master and develop via GitHub pull requests.
```
Comment (PaulineMauryL) [inline docs/source/github_workflow.rst:16]: I think this part can be added to Contributing.md

Comment (PaulineMauryL) [reply docs/source/github_workflow.rst:16]: even the whole file is interesting to put in.. could be top level contributing.md. not sure what is best, any idea ?

File: docs/source/github_workflow.rst [hunk 2]
```diff
@@ -1,37 +1,91 @@
...
+      - Yes
+      - Yes
+      - No
+      - No
+      - No
+      - No
+    * - Docker build and push
+      - No
+      - No
+      - Yes (tags git sha)
```
Comment (PaulineMauryL) [inline docs/source/github_workflow.rst:46]: what does tags git and sha mean ?

File: docs/source/github_workflow.rst [hunk 3]
```diff
@@ -1,37 +1,91 @@
...
+      - No
+      - Yes (tags git sha)
+      - Yes (tags latest and semver)
```
Comment (PaulineMauryL) [inline docs/source/github_workflow.rst:49]: what is a semver tag ?

File: docs/source/github_workflow.rst [hunk 4]
```diff
@@ -1,37 +1,91 @@
...
+      - No
+      - No
+      - No
+      - Yes (must manually adjust version)
+      - No
+      - No
+    * - Documentation push
+      - No
+      - No
+      - Yes
```
Comment (PaulineMauryL) [inline docs/source/github_workflow.rst:67]: and so latests version ?

File: docs/source/github_workflow.rst [hunk 5]
```diff
@@ -1,37 +1,91 @@
...
+      - No
+      - No
+      - Yes (must manually add version)
```
Comment (PaulineMauryL) [inline docs/source/github_workflow.rst:70]: and so 'stable' version

**Review by PaulineMauryL — APPROVED** [review]
approved with a few addition to the doc if possible

File: docs/source/github_workflow.rst (hunk 2 above)
Comment (damienbfs) [reply docs/source/github_workflow.rst:46]: The tag of the dockerfile is the git commit sha. I'll try to make it clearer without it taking too much space.

File: docs/source/github_workflow.rst (hunk 4 above)
Comment (damienbfs) [reply docs/source/github_workflow.rst:67]: yes

File: docs/source/github_workflow.rst (hunk 5 above)
Comment (damienbfs) [reply docs/source/github_workflow.rst:70]: yes

File: docs/source/github_workflow.rst (hunk 1 above)
Comment (damienbfs) [reply docs/source/github_workflow.rst]: Moved to top-level contributing.md

**Pull request merged**